### ISSUE TITLE
missing dependencies are not installed

### ISSUE THREAD

**Issue by Morgy93** [dermatz/ddev-woodoo-buildtools-magento#21]
In my case `@hyva-themes/hyva-modules` was missing and I got an error during theme build.

I had to manually install the dependencies and it worked flawless afterwards.

**Comment by Morgy93** [comment]
Similar:
If there is no `grunt` in your package.json, then it does not work as well:
```
Aborted due to warnings.


✖ Grunt task failed. Try to repair it for you.



removed 100 packages, and audited 9 packages in 487ms

found 0 vulnerabilities
.ddev/commands/web/woodoo_components/build: line 55: ./node_modules/.bin/grunt: No such file or directory
.ddev/commands/web/woodoo_components/build: line 56: ./node_modules/.bin/grunt: No such file or directory




  ✖ Grunt task failed again. Could not repair.
```

**Comment by dermatz** [comment]
@Morgy93 this should now be fixed with version 1.4.1.
